nutrition_labels/grant_tagger.py

```diff
@@ -324,9 +330,6 @@ def save_model(self, output_path, evaluation_results=None):
             pickle.dump(self.model, f)
         with open(os.path.join(output_path, "vectorizer.pickle"), "wb") as f:
             pickle.dump(self.vectorizer, f)
-        if self.scaler:
```

Comment:
not that you can pickle the entire pipeline, you don't need to pickle components separately